tests/unit/api/dashboard/test_search_dashboard_api.py

```diff
@@ -48,18 +48,6 @@ def test_should_give_empty_result_when_there_are_no_results_from_proxy(self) ->
         }
         self.assertEqual(response.json, expected_response)
 
-    def test_should_get_default_response_values_when_values_not_in_proxy_response(self) -> None:
-        self.mock_proxy.fetch_dashboard_search_results.return_value = \
-            SearchResult(total_results=1, results=[{}])
```

Comment:
Removing this test case for `Dashboard` model fails to instantiate without params.